tests/rptest/chaos_tests/single_fault_test.py

```diff
@@ -60,6 +68,43 @@ def run(self,
                 # if it failed, the check will be performed automatically
                 self.redpanda.raise_on_crash()
 
+    def _transfer_leadership(self,
+                             topic,
+                             target_id,
+                             partition=0,
+                             namespace="kafka",
+                             leader_id=None,
```

Comment:
Is it just an optimization when we already know the current leader?
nit: rename the parameter somehow to indicate it's the current leader?

Reply:
> Is it just an optimization when we already know the current leader?

correct